include/swift/ClangImporter/ClangImporter.h

```diff
@@ -730,6 +730,8 @@ ClangInvocationFileMapping getClangInvocationFileMapping(
     ASTContext &ctx,
     llvm::IntrusiveRefCntPtr<llvm::vfs::FileSystem> vfs = nullptr);
 
+bool hasNonEscapableAttr(const clang::RecordDecl *decl);
```

Comment:
Minor: Does this need to be exposed as API of ClangImporter?

Reply:
Hmm, actually I could move this to `ImporterImpl.h`. What do you think?

Reply:
Yeah, I think that would be slightly better, up to you though.

Reply:
Looks like there's already a precedent: `bool importer::hasUnsafeAPIAttr(const clang::Decl *decl)`

Reply:
The new version follows the same pattern as `hasUnsafeAPIAttr`.